acceptance/analyzer_test.go

```diff
@@ -27,36 +27,28 @@ import (
 	h "github.com/buildpacks/lifecycle/testhelpers"
 )
 
-const (
-	readWriteImage = "image-readable-writable"
-	onlyReadImage  = "image-readable"
-	onlyWriteImage = "image-writable"
-	noAccessImage  = "noAccessImage"
-)
-
 var (
 	analyzerBinaryDir    = filepath.Join("testdata", "analyzer", "analyze-image", "container", "cnb", "lifecycle")
 	analyzeDockerContext = filepath.Join("testdata", "analyzer", "analyze-image")
 	analyzeImage         = "lifecycle/acceptance/analyzer"
 	analyzerPath         = "/cnb/lifecycle/analyzer"
 	cacheFixtureDir      = filepath.Join("testdata", "analyzer", "cache-dir")
 	daemonOS, daemonArch string
-	readOnlyRegistry     *ih.DockerRegistry
-	authRegistry         *ih.DockerRegistry
-	customRegistry       *ih.DockerRegistry
+	testRegistry         *ih.DockerRegistry
 	registryNetwork      string
 	fixtures             analyzeFixtures
 )
 
 type analyzeFixtures struct {
-	authRegAppImage       string
+	authRegAppImage       string // read and write access
 	authRegAuthConfig     string
-	authRegCacheImage     string
-	authRegOtherAppImage  string
+	authRegCacheImage     string // read and write access
+	authRegOtherAppImage  string // read and write access
 	daemonAppImage        string
 	daemonCacheImage      string
-	readOnlyRegAppImage   string
-	readOnlyRegCacheImage string
+	readOnlyRegAppImage   string // read-only
+	readOnlyRegCacheImage string // read-only
+	inaccessibleImage     string // no access at all
```

Comment:
I think the comments are redundant.

Reply:
I refactored the names in the last [commit](https://github.com/buildpacks/lifecycle/pull/685/commits/59c111c24edd1c3bb2cdcaba6ba521a5b3718d4d)